app/uk/gov/hmrc/calculatenifrontend/controllers/TablesController.scala

```diff
@@ -102,17 +103,27 @@ class TablesController @Inject()(
             "Share Fishing Weekly Rate" -> vdw.toString,
             "Share Fishing Total" -> (vdw * data.noOfWeeks).toString
           )
-        }) ++ List (
-//          "Date Late For Short Term Benefits (STB)" -> "???",
-          "Final Date For Payment" -> (lowerBound, data).getFinalDate.value.ukFormat,
+        }) ++ (if (voluntaryYearSelect.contains(lowerBound.getYear)) {
```

Comment:
Here you insert the word "Voluntary" into the title for the relevant years or not, better to make that clear rather than duplicate the whole entry dependent on the chosen year.

Reply:
Walked this through with Ben - but if you think there is a better way or are we talking a comment ?

Reply:
Refactored